js/scripts/zone_loaders/village_zone_loaders.js

```diff
@@ -214,7 +214,9 @@ const loadVillageMain = () => {
     };
 
     const addEntities = () => {
-        ASSET_MGR.playMusic(MUSIC.PEACEFUL_CHIPTUNE.path, MUSIC.PEACEFUL_CHIPTUNE.volume);
+        setTimeout(() => {
```

Comment:
Why is there a timeout here?

Reply:
If it's empty, then that's a mistake. I was using it originally to make music play later on as opposed to immediately when the village was loaded.